mods/ts/mod.yaml

```diff
@@ -396,3 +396,56 @@ ModContent:
 					^Content/ts/temperat.mix:
 						Offset: 73056940
 						Length: 2037606
+		origin: C&C The Ultimate Collection (Origin version)
```

Comment:
Origin seems to automatically pick the localized version at least during the on the house promotion and I am not sure it allows you to download original locales like Steam does:
![image](https://cloud.githubusercontent.com/assets/756669/16363414/8051a7f8-3bc9-11e6-818b-0f119b71448d.png)